install/Knative-with-ICP.md

```diff
@@ -0,0 +1,185 @@
+ibm-privileged-psp          true      *                                                                                                                     RunAsAny   RunAsAny           RunAsAny    RunAsAny    false            *
+ibm-restricted-psp          false                                                                                                                           RunAsAny   MustRunAsNonRoot   MustRunAs   MustRunAs   false            configMap,emptyDir,projected,secret,downwardAPI,persistentVolumeClaim
+```
+
+2. Create a cluster role for the pod security policy resource. The resourceNames for this role must be the name of the pod security policy that was created previous. Here we use `ibm-privileged-psp`. Run the following command:
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+ name: knative-role
+rules:
+ -
+   apiGroups:
+     - extensions
+   resourceNames:
+     - ibm-privileged-psp
+   resources:
+     - podsecuritypolicies
+   verbs:
+     - use
+EOF
+```
+
+3. Set up cluster role binding for the service account in Knative namespace. By using this role binding, you can set the service accounts in the namespace to use the pod security policy that you created.
+```shell
+cat <<EOF | kubectl apply -f - 
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+ name: knative-serving-psp-users
+roleRef:
+ apiGroup: rbac.authorization.k8s.io
+ kind: ClusterRole
+ name: knative-role
+subjects:
+ -
+   apiGroup: rbac.authorization.k8s.io
+   kind: Group
+   name: "system:serviceaccounts:knative-serving"
+EOF
+```
+
+Use the same method to add the other Knative namespaces to the `ibm-privileged-psp` pod security policy.
+
+Monitor the Knative components until all of the components show a `STATUS` of
+`Running`:
+
+```
+kubectl get pods --namespace knative-serving
+kubectl get pods --namespace knative-build
+```
+
+Just as with the Istio components, it will take a few seconds for the Knative
+components to be up and running; you can rerun the command to see the current status.
+
+> Note: Instead of rerunning the command, you can add `--watch` to the above
+  command to view the component's status updates in real time. Use CTRL+C to exit watch mode.
+
+Now you can deploy an app to your newly created Knative cluster.
+
+## Deploying an app
+
+Now that your cluster has Knative installed, you're ready to deploy an app.
+
+If you'd like to follow a step-by-step guide for deploying your first app on
+Knative, check out the
+[Getting Started with Knative App Deployment](getting-started-knative-app.md)
+guide.
+
+If you'd like to view the available sample apps and deploy one of your choosing,
+head to the [sample apps](../serving/samples/README.md) repo.
+
+> Note: When looking up the IP address to use for accessing your app, you need to look up
```

Comment:
The markdown note and codeblock are disconnected here. Maybe just use bold?
```suggestion
*Note*: When looking up the IP address to use for accessing your app, you need to look up
```

Reply:
Done